🐛 Bug: `npx gravlax` does not work

### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/danvk/gravlax/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

I should be able to run `npx gravlax`

### Actual

```
$ npx gravlax
npm ERR! could not determine executable to run

npm ERR! A complete log of this run can be found in: /Users/danvk/.npm/_logs/2024-02-02T23_05_38_049Z-debug-0.log
```

### Additional Info

This command is in README.md so it should work!

--- a/package.json
+++ b/package.json
@@ -13,6 +13,7 @@
 	},
 	"type": "module",
 	"main": "./lib/index.js",
+	"bin": "./lib/index.js",
 	"files": [
 		"lib/",
 		"package.json",

